src/target/riscv/riscv-013.c

```diff
@@ -875,8 +877,11 @@ static uint32_t access_register_command(struct target *target, uint32_t number,
 	return command;
 }
 
-static int register_read_abstract(struct target *target, uint64_t *value,
-		uint32_t number, unsigned size)
+/**
+ * This function is used explicitly to determine xlen during examine.
+ */
```

Comment:
I'd recommend to drop this comment. That's because this function is used also for all abstract reads (indirectly from `register_read_abstract`), not just exclusively during examine.

```suggestion
```

Reply:
Addressed.